src/main/java/io/netty/incubator/codec/http3/Http3SettingsFrame.java

```diff
@@ -51,6 +51,18 @@ default long type() {
      */
     Long get(long key);
 
+    /**
+     * Get a setting from the frame.
+     *
+     * @param key   the key of the setting.
+     * @param defaultValue If the setting does not exist.
+     * @return the value of the setting or {@code defaultValue} if none was found with the given key.
+     */
+    default long getOrDefault(long key, long defaultValue) {
```

Comment:
btw I wondered if we should just make this `Long` as this way it maybe be easier for people to return `null` and so better understand if it was really there.